keps/sig-node/4742-node-topology-downward-api/README.md

```diff
@@ -402,6 +405,7 @@ Revert feature gate and stop consuming downward API.
 ## Implementation History
 
 - `v1.33`: initial KEP is accepeted and alpha implementation is complete
+- `v1.34`: fix topology labels from topology.k8s.io to topology.kubernetes.io. Promote feature gate to Beta.
```

Comment:
this is the fix in the KEP docs, not in actual code, correct?

Reply:
cc @munnerz to confirm